### PR TITLE
Added Bootstrap option

### PR DESCRIPTION
**Pull request by AndreyAzimov**
Run: `rails css:install:bootstrap`

### CODE REVIEW

**Comment by leastbad** [comment]
I think that this is awesome.

File: lib/install/bootstrap/install.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+say "Install Bootstrap 5 with Popperjs/core"
+copy_file "#{__dir__}/bootstrap_js_files.js", "bootstrap_js_files.js"
+copy_file "#{__dir__}/application.scss", "app/assets/stylesheets/application.scss"
+
+run "yarn add bootstrap@5.1.1"
```
Comment (DavidColby) [inline lib/install/bootstrap/install.rb:5]: I haven't used Bootstrap in quite a while but is using specific version numbers the right call here?

Seems like they'd need to be bumped regularly vs. just using @latest instead.

File: lib/install/bootstrap/install.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+say "Install Bootstrap 5 with Popperjs/core"
+copy_file "#{__dir__}/bootstrap_js_files.js", "bootstrap_js_files.js"
```
Comment (marcoroth) [inline lib/install/bootstrap/install.rb:2]: Is it just me or does the filename `bootstrap_js_files.js` feel weird? 

I would have expected something like `bootstrap.js`.

```suggestion
copy_file "#{__dir__}/bootstrap_js_files.js", "bootstrap.js"
```

File: lib/install/bootstrap/install.rb (hunk 1 above)
Comment (AndreyAzimov) [reply lib/install/bootstrap/install.rb]: @DavidColby agree! fixed!

File: lib/install/bootstrap/install.rb (hunk 2 above)
Comment (AndreyAzimov) [reply lib/install/bootstrap/install.rb]: @marcoroth agree! I renamed and pushed the changes to the current fork.

**Comment by gczh** [comment]
Awesome stuff @AndreyAzimov 🍻

**Comment by defsdoor** [comment]
Is this for webpacker use only  ?

Is it strange that there are specific framework options but only when using webpacker - especially when this is (rightly) not the default method now.

sass bootstrap support with esm js imports would be a nice option (although pretty trivial to do manually)

**Comment by dhh** [comment]
No, this does not require Webpacker. You can use it with either importmaps or jsbundling-rails.

**Comment by defsdoor** [comment]
aah - I saw yarn being used and assumed it was webpacking - is yarn a base requirement for rails now ?

**Comment by dhh** [comment]
Nope. Just a requiring for using node-based bundling tools.

**Comment by jamesez** [comment]
I found bootstrap 5 needs to go through autoprefixer or some styles don't work. I'm dragging in `postcss` and running `postcss  --use autoprefixer`

**Pull request merged**